benchmarks/big.lua

```diff
@@ -0,0 +1 @@
+../third-party/lua/testes/big.lua
```

Comment:
This doesn't look like a valid Lua source file to me (nor do the two below)?

Reply:
Bit of Linux black magic. I symlinked these tests (I just call benchmarks) to their respective locations in `./tests` and `./third_party/lua` (provided submodules are initialised). I checked and they link correctly on a fresh clone. I think this is cleaner, but if you think a direct copy is better, perhaps to pin in case of any future changes, then I'll do that.

Reply:
Ah!